### ISSUE TITLE
Support for dialog windows via the command line

### ISSUE THREAD

**Issue by joewing** [joewing/jwm#249]
JWM currently supports confirmation dialogs to confirm exit and killing windows.  As mentioned in issue #83, it would be nice to be able to re-use these for other things via the command line.

For example:
```
jwm -dialog "some message here"
```

The return status from the command could indicate the selection (perhaps 0 for OK, 1 for cancel, 2 for error).

**Comment by dumblob** [comment]
Hm, smells like more and more widget-like code will be needed. I think the time has come to take a look e.g. at [Zahnrad](https://github.com/vurtun/zahnrad). Also because of the future support for Wayland as mentioned in https://github.com/joewing/jwm/issues/103 .

**Comment by 01micko** [comment]
I would think the less widget like code the better (KISS)  but it would be good if a `-dialog` option could support an icon scaled at a fixed size (24x24?) and perhaps get its colors/properties from the `<WindowStyle>` tag.

**Comment by Israel-** [comment]
I agree with @01micko a fixed sized icon would be ok, or at least an icon scaled to a percent of the dialog screen, something perhaps like 1/3 the height (for both w/h) seems about the general size for a dialog icon, this would make things work better on different resolutions, if calculating pixels.... unless there is code for this sort of thing in JWM somewhere.

The alternative might be to make a JWM library, and we could play around with making a dialog program, perhaps?

**Comment by dumblob** [comment]
@Israel- all what you mentioned is solved in [Zahnrad](https://github.com/vurtun/zahnrad) which is prepared exactly for such cases, when one needs to draw something more than just a straight line, but doesn't want to introduce an external dependency (Zahrad is not a library, but only 2 drop-in files - `zahnrad.c` and `zahnrad.h`).

By the way, using Zahnrad would allow JWM to become more lean (get rid of rendering code, special graphical features etc.), but support more features in the future like bitmap decorations anywhere (e.g. instead of plain gradients in menus). This is definitely not thinkable with hard-coding such support directly to JWM.

>The alternative might be to make a JWM library, and we could play around with making a dialog program, perhaps?

I would discourage from doing this. First, there are number of other dialog programs out there. Second, it's already quite out of scope JWM (as an enhanced **window** manager). Third, the maintenance cost is not worth it in my opinion.

But of course, everything I mentioned depends on @joewing and his vision of the future direction of JWM.
